api/src/main/java/io/opentelemetry/metrics/CounterDouble.java

```diff
@@ -36,34 +36,37 @@
  *           .setUnit("1")
  *           .setLabelKeys(Collections.singletonList("Key"))
  *           .build();
- *   // It is recommended to keep a reference of a Handle.
- *   private static final CounterDouble.Handle someWorkHandle =
- *       counter.getHandle(Collections.singletonList("SomeWork"));
+ *   // It is recommended to keep a reference of a Bound.
+ *   private static final BoundDoubleCounter someWorkBound =
+ *       counter.getBound(Collections.singletonList("SomeWork"));
```

Comment:
this method should be `bind` by the OTEP:  https://github.com/open-telemetry/oteps/blob/master/text/0070-metric-bound-instrument.md#internal-details

Reply:
Will do in a separate PR. Need to merge this because it got large.